src/foam/u2/filter/FilterView.js

```diff
@@ -321,6 +325,7 @@ foam.CLASS({
             .end()
           .end()
           .start()
+            .addClass(this.myClass('hide-overflow'))
```

Comment:
given it is so directly mapped to a particular CSS attribute, you could just do .style({overflow: 'hidden'}) instead of addClass() and the defining the CSS for that class.

Reply:
fixed